Language/Generics/Superbounded_types/class_A01_t05.dart

```diff
@@ -2,26 +2,35 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-/// @assertion It is a compile-time error if a parameterized type [T] is
+/// @assertion It is a compile-time error if a parameterized type `T` is
 /// super-bounded when it is used in any of the following ways:
-///   [T] is an immediate subterm of a new expression (16.15.1) or a constant
-///   object expression
-/// @description Checks that compile error is thrown when not well-bounded
-/// parametrized type is used in the constant object expression with [is]
-/// constructions.
+/// - `T` is an immediate subterm of a new expression or a constant object
+///   expression.
+///
+/// @description Checks that it is a compile-time error if a parametrized
+/// super-bounded type is used in a constant `is` expression and not a top type.
```

Comment:
What we're checking here is that a type which is not well-bounded is an error as the right hand operand of `is`. Of course, it would be an error in every other location as well.

There's no great need to mention 'super-bounded types', because there aren't any of them here. Perhaps the point is that "this is not a super-bounded type, and we want the compiler to say so"? Not sure how to say that in a way that helps the reader more than a plain "this type is an error because it isn't well-bounded".

Finally, it doesn't make much sense to talk about a _parameterized_ super-bounded type because super-bounded types are _always_ parameterized.

Perhaps something like this could be used:

Checks that it is a compile-time error if a type which is not well-bounded
is used as the right hand operand in an `is` expression.